lib/testLauncher/SuitestLauncher.js

```diff
@@ -59,7 +59,17 @@ class SuitestLauncher {
 						},
 					};
 
-					log.startOnDevice(device.deviceId, this.restArgs.join(' '));
+					let ws = null;
+
+					try {
+						ws = log.startOnDevice(device.deviceId, this.restArgs.join(' '), logDir);
+					} catch (e) {
```

Comment:
@andriichumak , @JeKachan, @AntonSBilan  I'm thinking whether we should exit process with 1 code if writing to logs file was unsuccessful or just log some warning?

Reply:
I prefer warning message. Or we should specify another error code for this case (on my point of view we should avoid using 1 for all errors)

Reply:
I think exiting with error is the correct behaviour. Error message should clearly state that tests were aborted due to lack of write permission in the logs folder.